src/templates/api/confirmation_result.html

```diff
@@ -0,0 +1,17 @@
+{% set no_login = True %}
+{% extends "layout.html" %}
+{% block content %}
+    <div class="row justify-content-center">
+        <div class="col-sm-6">
+            <div class="card">
+                <div class="card-body">
+                    <p>You can now close this window and go back to the app.</p>
+                    <button type="submit" class="btn btn-primary" id="close">Close</button>
```

Comment:
I think it would be good to center this(horizontally). because it looks like its out of place being on the left hand side

Reply:
fix off center issues